### ISSUE TITLE
Is there any paper about this data?

### ISSUE THREAD

**Issue by wangliangguo** [harvardnlp/sent-summary#1]
Is there any paper that report there performance on this dataset?

**Comment by srush** [comment]
Yes, A Neural Attention Model for Abstractive Sentence Summarization , Rush
et al http://aclweb.org/anthology/D15-1044

**Comment by ailsamm** [comment]
@srush So the data is the 'pruned' version of Gigaword as created in the linked paper? Thanks.
